docs/hooks/use-fetcher.md

```diff
@@ -70,7 +70,7 @@ The `formData` can be multiple types:
 
 - [`FormData`][form_data] - A `FormData` instance.
 - [`HTMLFormElement`][html_form_element] - A [`<form>`][form_element] DOM element.
-- `Object` - An object of key/value pairs that will be converted to a `FormData` instance.
+- `Object` - An object of key/value pairs that will be converted to a `FormData` instance. It can be any JSON data if the option `encType: "application/json"` is used. See [`useSubmit`][use-submit] for more details.
```

Comment:
```suggestion
- `Object` - An object of key/value pairs that will be converted to a `FormData` instance by default.  You can pass a more complex object and serialize it as JSON by specifying `encType: "application/json"`. See [`useSubmit`][use-submit] for more details.
```

Reply:
It might also be good to enhance the code examples below?


```tsx
// Submit a FormData instance (GET request)
let formData = new FormData();
fetcher.submit(formData);

// Submit the HTML form element
fetcher.submit(event.currentTarget.form, {
  method: "POST",
});

// Submit key/value JSON as a FormData instance
fetcher.submit(
  { serialized: "values" },
  { method: "POST" }
);

// Submit raw JSON
fetcher.submit(
  { 
    deeply: { 
      nested: { 
        json: "values"
      }
    }
  },
  { 
    method: "POST", 
    encType: "application/json"
  }
);
```

Reply:
Applied the change and added the code examples.

Reply:
Hmm, what went wrong. The another commit containing the code examples did not get in to this PR.

Reply:
OK, it was just slow.